code/src/components/ThoughtForm.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+
+import './Styles/thoughtform.css';
+
+export const ThoughtForm = ({onThoughtChange}) => {
+
+    //State hooks for new thoughts
+    const [newThought, setNewThought] = useState(''); 
+
+    // Function for handling input and posting of new happy thought
+    const handleSubmit = event => {
+        event.preventDefault();
+        onThoughtChange(newThought);
+        setNewThought('')
+    }
+
+    // Form for posting happy thoughts
+    return (
+        <form onSubmit={handleSubmit} className='thoughts-form'>
+            <h2 className='thoughts-form__header'> 
+                What's making you happy right now?
+            </h2>
+                <input
+                    type='text'
```

Comment:
when I input a long message in your form it only uses the first row. So when I keep typing I cannot see my whole message. Maybe try a <textarea> instead of < input type=text > or if the css property word-break: break-word works on this as well :)